language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/contexts/RenameContextImpl.java

```diff
@@ -61,4 +63,21 @@ public RenameContext build() {
             return this;
         }
     }
+
+    @Override
+    public void setHonorsChangeAnnotations(Boolean honorsChangeAnnotations) {
+        if (honorsChangeAnnotations == null) {
```

Comment:
One suggestion for the future reference to avoid if-clause, you could use this for these cases - Boolean.TRUE.equals(honorsChangeAnnotations);

Reply:
+1. done